.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: deploy
+
+on:
+  push:
+    branches-ignore:
+      - '**'
```

Comment:
If you really don't want to run the workflow when pushing to a new branch, isn’t it enough to just get rid of these two lines? Am I missing something?